ElephantVentures/cheetah-vision-poc
# Added audio and video reference

## Ticket
- N/A

## Dependencies
- N/A

## Changes
- Added video immersive view struct
- Added audio player manager class
- Added a new video, assets, and test audios

## Pre-Test Plan
- N/A

## Test Plan
1. Pull down branch
2. Run the Simulator in Xcode
3. Proceed to the app
4. Click on any of the immersive images/video
- [ ] Verify that the sound works
- [ ] Verify that the video ambiance works with looping


## Resources
- N/A
